config/chassis_modules.py

```diff
@@ -27,7 +28,10 @@ def get_config_module_state(db, chassis_module_name):
     config_db = db.cfgdb
     fvs = config_db.get_entry('CHASSIS_MODULE', chassis_module_name)
     if not fvs:
-        return 'up'
+        if is_smartswitch():
+            return 'down'
+        else:
```

Comment:
since code is common for NPU and DPU, do you need to check for is_dpu() as well?

Reply:
@vvolam No, on the DPUs this config should be a don't care.